pkg/cmd/addon/util.go

```diff
@@ -40,3 +49,33 @@ func getAddonVersion(addon *extensionsv1alpha1.Addon) string {
 	}
 	return ""
 }
+
+func CheckBeforeDisableAddon(f cmdutil.Factory, addons []string) error {
```

Comment:
This function is used by `disable` and `uninstall`, so its name is not appropriate.

Maybe `CheckAddonUsedByCluster`